front/lib/models/assistant/actions/browse.ts

```diff
@@ -0,0 +1,168 @@
+import type { BrowseActionOutputType } from "@dust-tt/types";
+import type {
+  CreationOptional,
+  ForeignKey,
+  InferAttributes,
+  InferCreationAttributes,
+} from "sequelize";
+import { DataTypes, Model } from "sequelize";
+
+import { AgentConfiguration } from "@app/lib/models/assistant/agent";
+import { AgentMessage } from "@app/lib/models/assistant/conversation";
+import { frontSequelize } from "@app/lib/resources/storage";
+
+export class AgentBrowseConfiguration extends Model<
+  InferAttributes<AgentBrowseConfiguration>,
+  InferCreationAttributes<AgentBrowseConfiguration>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare agentConfigurationId: ForeignKey<AgentConfiguration["id"]>;
+
+  declare sId: string;
+
+  declare name: string | null;
+  declare description: string | null;
+}
+
+AgentBrowseConfiguration.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    sId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    description: {
+      type: DataTypes.TEXT,
+      allowNull: true,
+    },
+  },
+  {
+    modelName: "agent_browse_configuration",
+    indexes: [
+      {
+        unique: true,
+        fields: ["sId"],
+      },
+      {
+        fields: ["agentConfigurationId"],
+        concurrently: true,
+      },
+    ],
+    sequelize: frontSequelize,
+  }
+);
+
+AgentConfiguration.hasMany(AgentBrowseConfiguration, {
+  foreignKey: { name: "agentConfigurationId", allowNull: false },
+});
+AgentBrowseConfiguration.belongsTo(AgentConfiguration, {
+  foreignKey: { name: "agentConfigurationId", allowNull: false },
+});
+
+export class AgentBrowseAction extends Model<
+  InferAttributes<AgentBrowseAction>,
+  InferCreationAttributes<AgentBrowseAction>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare browseConfigurationId: string;
+
+  declare urls: string;
```

Comment:
here we should do `string[]` IMHO, just like e.g. tags from [process action model](https://github.com/dust-tt/dust/blob/932b70f9409fb904cc5537f7d5ef5bd8bed82334/front/lib/models/assistant/actions/process.ts#L21)

Reply:
Nice, I did not know about postgres arrays